package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Did you `yarn add` from the project root instead of from 'frontend`?

Reply:
Deleting on https://github.com/streamlit/streamlit/pull/501, it was a mistake